src/layout/layout.js

```diff
@@ -401,13 +401,15 @@ function MaterialLayoutTab(tab, tabs, panels, layout) {
     }
 
     tab.addEventListener('click', function(e) {
-      e.preventDefault();
-      var href = tab.href.split('#')[1];
-      var panel = layout.content_.querySelector('#' + href);
-      layout.resetTabState_(tabs);
-      layout.resetPanelState_(panels);
-      tab.classList.add(layout.CssClasses_.IS_ACTIVE);
-      panel.classList.add(layout.CssClasses_.IS_ACTIVE);
+      if (tab.getAttribute('href').charAt(0) === '#') {
```

Comment:
@surma Such an anchor would work without reloading. Not sure how to handle this though, as there are different valid ways for the page to reference itself.

* index.html#something (page only)
* ../dir/index.html#something (relative urls)
* /dir/index.html#something (absolute urls)
* //example.com/dir/index.html#something (protocol-relative urls)
* http://example.com/dir/index.html#something (full url)
* ...?

Checking this in JS will be overkill.